test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -180,3 +269,24 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
       ;; just test that this bootstrap config file can be read successfully
       ;; (ie, this does not throw an exception)
       (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-url]))))
+
+(deftest duplicate-service-definitions
```

Comment:
I don't understand what this test is doing.  It doesn't appear to assert anything.  Should this test - and the associated `duplicate_services.cfg` - be part of the work that you're doing for [TK-211](https://tickets.puppetlabs.com/browse/TK-211)?